src/Kraken/Spot/API/AssetPair.jl

```diff
@@ -0,0 +1,155 @@
+module AssetPair
+
+export AssetPairQuery,
+    AssetPairData,
+    asset_pair
+
+using Serde
+using Dates, NanoDates, TimeZones
+
+using CryptoAPIs.Kraken
+using CryptoAPIs.Kraken: Data
+using CryptoAPIs: Maybe, APIsRequest
+
+@enum AssetPairInfo info leverage fees
+
+Base.@kwdef struct AssetPairQuery <: KrakenPublicQuery
+    pair::Maybe{String} = nothing
+    info::AssetPairInfo = info
+end
+
+struct Fee
+    volume::Int64
+    percent::Float64
+end
+
+struct AssetPairData <: KrakenData
+    _quote::String
+    aclass_base::String
+    aclass_quote::String
+    altname::String
+    base::String
+    cost_decimals::Int64
+    costmin::Maybe{Float64}
+    fee_volume_currency::String
+    fees::Maybe{Vector{Fee}}
+    fees_maker::Vector{Fee}
+    leverage_buy::Vector{Int64}
+    leverage_sell::Vector{Int64}
+    lot::String
+    lot_decimals::Int64
+    lot_multiplier::Int64
+    margin_call::Int64
+    margin_stop::Int64
+    ordermin::Maybe{Float64}
+    pair_decimals::Int64
+    status::String
+    tick_size::Maybe{Float64}
+    wsname::String
+end
+
+function Serde.custom_name(::Type{AssetPairData}, ::Val{:_quote})
+    return "quote"
+end
+
+struct FeeData <: KrakenData
+    fee_volume_currency::String
+    fees::Vector{Fee}
+    fees_maker::Vector{Fee}
+end
+
+struct LeverageData <: KrakenData
+    leverage_buy::Maybe{Vector{Int64}}
+    leverage_sell::Maybe{Vector{Int64}}
+end
```

Comment:
I suggest making the following renaming

- `AssetPairData` -> `AssetPairInfoData`
- `FeeData` -> `AssetPairFeeData`
- `LeverageData` -> `AssetPairLeverageData`